decidim-generators/lib/decidim/generators/app_templates/secrets.yml.erb

```
@@ -40,6 +40,7 @@ decidim_default: &decidim_default
   maximum_conversation_message_length: <%%= Decidim::Env.new("DECIDIM_MAXIMUM_CONVERSATION_MESSAGE_LENGTH", "1000").to_i %>
   password_blacklist: <%%= Decidim::Env.new("DECIDIM_PASSWORD_BLACKLIST").to_array(separator: ", ").to_json %>
   allow_open_redirects: <%%= Decidim::Env.new("DECIDIM_ALLOW_OPEN_REDIRECTS").to_boolean_string %>
+  service_worker_enabled: <%%= Decidim::Env.new("DECIDIM_SERVICE_WORKER_ENABLED").to_boolean_string %>
```

Could we add here as a default what you had before?
```suggestion
  service_worker_enabled: <%%= Decidim::Env.new("DECIDIM_SERVICE_WORKER_ENABLED", Rails.env.exclude?("development")).to_boolean_string %>
```

I actually found it useful that it defaults to `false` in the development environment and you can then enable it through the ENV var if you want to.